src/Components/Details/Details.css

```diff
@@ -0,0 +1,68 @@
+.details__details-view {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.return-to-home-view-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-family: 'Oswald',
+  sans-serif;
+  font-weight: 200;
+  font-stretch: normal;
+  font-size: 1.5rem;
+  color: white;
+  border: 0;
+  padding: .5rem;
+  margin: 2rem;
+  align-self: flex-start;
+}
+
+button:hover {
```

Comment:
Could this be specified as `.return-to-home-view-btn:hover` instead of `button:hover` ? Not sure if it matters, but seems a bit easier to follow when looking through the css!